M2/Macaulay2/packages/Varieties.m2

```diff
@@ -223,6 +287,25 @@ OO = new ScriptedFunctor from {
 OO.texMath = ///{\mathcal O}///
```

Comment:
Why does ```sheaf Ring``` assume that one wants to work with a projective variety?  Will caching solve this issue (couldn't one want to work with both the affine variety and the projective variety associated to one ring)?

Reply:
I agree, and in fact have added a comment about this above that method. I think we should discuss this, and what should `variety Ring` do, at the next meeting.